src/nb_jsonapi_client/common.py

```diff
@@ -99,18 +100,20 @@ def mark_invalid(self):
 
 def error_from_response(response_content):
     try:
-        error_str = response_content['errors'][0]['title']
+        # error_str = response_content['errors'][0]['title']
+        # error_str = f"{response_content['code']}: {response_content['message']}"
+        error_str = pprint.pformat(response_content)
```

Comment:
NationBuilder uses more of the error features than this library could handle originally.